go.mod

```diff
@@ -4,7 +4,6 @@ go 1.17
 
 require (
```

Comment:
Hmm, do we need a `go.mod` here?

Reply:
oh, this was messing up go fmt, but I guess it does belong in a different pr